### ISSUE TITLE
[BUG] After creating a C# test project (xunit, nunit), the project will no longer build

### ISSUE THREAD

**Issue by scaredcr6w** [microsoft/vscode-dotnettools#1169]
### Describe the Issue

After adding any type of test project (xunit, nunit) the project will no longer build and the following errors are showing up:

dotnet build ../vscode bug/TestProject1/TestProject1.csproj /property:GenerateFullPaths=true /consoleloggerparameters:NoSummary
MSBuild version 17.9.8+b34f75857 for .NET
Determining projects to restore...
All projects are up-to-date for restore.
../vscode bug/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs(4,12): error CS0579: Duplicate 'global::System.Runtime.Versioning.TargetFrameworkAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(13,12): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(14,12): error CS0579: Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(15,12): error CS0579: Duplicate 'System.Reflection.AssemblyFileVersionAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(16,12): error CS0579: Duplicate 'System.Reflection.AssemblyInformationalVersionAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(17,12): error CS0579: Duplicate 'System.Reflection.AssemblyProductAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(18,12): error CS0579: Duplicate 'System.Reflection.AssemblyTitleAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/obj/Debug/net8.0/vscode bug.AssemblyInfo.cs(19,12): error CS0579: Duplicate 'System.Reflection.AssemblyVersionAttribute' attribute [../vscode bug/vscode bug.csproj]
../vscode bug/TestProject1/obj/Debug/net8.0/TestProject1.GlobalUsings.g.cs(9,22): error CS0400: The type or namespace name 'Xunit' could not be found in the global namespace (are you missing an assembly reference?) [../vscode bug/vscode bug.csproj]
../vscode bug/TestProject1/UnitTest1.cs(6,6): error CS0246: The type or namespace name 'FactAttribute' could not be found (are you missing a using directive or an assembly reference?) [../vscode bug/vscode bug.csproj]
../vscode bug/TestProject1/UnitTest1.cs(6,6): error CS0246: The type or namespace name 'Fact' could not be found (are you missing a using directive or an assembly reference?) [../vscode bug/vscode bug.csproj]

### Steps To Reproduce

1. Create a console application using 'dotnet new console --use-program-main'
2. To the solution, add a new xunit test project
3. Use 'dotnet build'

### Environment Information

- OS: macOS 14.4.1 (23E224)
- VSCode: 1.89.1

**Comment by Sleepwalker423** [comment]
Same exact issue here.

**Comment by ocallesp** [comment]
Which version of C# Dev Kit extension are you using ?

**Comment by scaredcr6w** [comment]
> Which version of C# Dev Kit extension are you using ?

I'm using 1.7.27

**Comment by ocallesp** [comment]
On macOS, when using C# Dev Kit version 1.8.6 (pre-release) I found that creating a simple standalone xUnit project initializes the project system provider, but it fails to detect any containers or projects (DLLs)

**Comment by ocallesp** [comment]
@scaredcr6w  could you please share a project that reproduces the issue ?

**Comment by scaredcr6w** [comment]
https://github.com/scaredcr6w/csharpTestsBug

**Comment by chrisdel101** [comment]
I am facing this issue as well. If I create a new test proj alone it works, but I cannot add it to my project. I even created a new proj just to test this and it fails.
I've followed this exactly https://code.visualstudio.com/docs/csharp/testing#_enable-testing-and-adding-test-framework-packages-to-your-project.
No matter which framework I use the project falls apart. I even get `The type or namespace name 'NUnit' could not be found (are you missing a using directive or an assembly reference?)` which is impossible, since previous steps require it to be installed, and that succeeds, so it's 100% installed, and the proj files are all the defaults created using that link.

**Comment by snnrmn** [comment]
Here's the solution: https://github.com/dotnet/core/issues/4837

**Comment by chrisdel101** [comment]
Yes, this is the solution. For me there was no bug (face palm emoji). Tests cannot be nested but must be on same level as project folder.
